README.md

```diff
@@ -4,227 +4,194 @@
+the `openshift` directory must be created before.
 
-You can test users of Apisonator API services by starting its dependencies and
-then apisonator itself:
+Once this has been performed, go to the `openshift` directory and execute
+one of the available Makefile commands to run Apisonator components:
 
-> `$ script/test_external`
+#### Apisonator Listener
 
-This will launch the application and wait for requests. You could now ie. launch
-`3scale_core`'s testing against this backend instance.
+Execute the Apisonator Listener, exposing the port 3001:
 
-You can hit CTRL+C at any time to exit. If you wanted to pass extra parameters
-for the launcher, such as daemonizing, you could run this:
+```
+make listener
+```
 
-> `$ script/test_external -- -d`
+#### Apisonator Worker
 
-## Running
+```
+make worker
+```
 
-Apisonator has its own application runner for the listener service:
+#### Apisonator failed jobs rescheduler
 
-`bundle exec 3scale_backend help`
+Execute the 'cron' Apisonator component:
 
-That will give you a help message so that you figure out how to invoke it. Some
-advanced usage patterns are available, as well as support for multiple
-application servers. The usual invocation would look something like:
+```
+make cron
+```
 
-> `$ bundle exec 3scale_backend start -p 3001`
+#### Apisonator bash shell
 
-That would start the listener service in port 3001. There are however additional
-commands and flags that are worth knowing, so please take a look at the help
-message. An interesting command would be this one:
+Execute a bash shell with the Apisonator source code with all the available
+components:
 
-> `$ bundle exec 3scale_backend -s puma -X "-w 3" start -p 3001`
+```
+make bash
+```
 
-That would be identical to the command above, except it would force the server
-to be Puma with 3 workers. You can obtain more info about the server specific
-options with:
+### Manual execution
 
-> `$ bundle exec 3scale_backend -s puma help-server`
+Another way of executing the Apisonator components is by running a container
+using the previously generated Apisonator image:
 
-Additional information which is really important to how the application runs can
-be found in the manifest:
+#### Apisonator Listener
 
-> `$ bundle exec 3scale_backend manifest`
+To run an Apisonator listener, the script bin/3scale_backend is used. To
+run it from a previously generated Apisonator docker image:
 
-## Documentation
+```
+docker run -it amp:apisonator-<version_number> -p 3001:3001 --env-file <myenv_file> 3scale_backend start -p 3001 -x /dev/stdout
+```
 
-Make sure to read the corresponding documentation in [ActiveDocs](https://support.3scale.net/reference/active-docs) for the external
-API. You can also generate the documentation of the Internal API with Rake tasks.
+You can see all the available options of the apisonator listener by executing:
 
-## Integration flow
+```
+docker run -it amp:apisonator-<version_number> 3scale_backend --help
```

Comment:
just `help` is a better form, as it is a command that can receive other commands as argument.

Reply:
:+1: